package.json

```
@@ -24,7 +24,8 @@
     "react-i18next": "11.18.6",
     "react-rewards": "^2.0.4",
     "swr": "2.0.3",
-    "use-local-storage-state": "18.1.1"
+    "use-local-storage-state": "18.1.1",
+    "zod": "^3.21.4"
```

```suggestion
    "zod": "3.21.4"
```

renovate使っているので、package.json記載のバージョンは固定できると良いかなと思います

---

https://github.com/GoCon/2023/pull/184 にて対応しました 🙏